src/components/chat/ChatList.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, HStack, Image, StackDivider, VStack } from '@chakra-ui/react';
+import StatusIndicator from '@/components/common/StatusIndicator';
+import { useEffect, useState } from 'react';
+
+const CHATS: {
+  profileImg: string;
+  username: string;
+  status: 'ONLINE' | 'AWAY' | 'OFFLINE';
+  topMsg: string;
+  selected: boolean;
+}[] = [
+  {
+    profileImg: 'https://ddragon.leagueoflegends.com/cdn/13.12.1/img/profileicon/0.png',
+    username: 'hide on bush',
+    status: 'ONLINE',
+    topMsg: '자리있나요?',
+    selected: true,
+  },
+  {
+    profileImg: 'https://ddragon.leagueoflegends.com/cdn/13.12.1/img/profileicon/0.png',
+    username: 'hide on bush',
+    status: 'OFFLINE',
+    topMsg: '자리있나요?',
+    selected: false,
+  },
+];
+
+interface ChatProps {
+  profileImg: string;
+  username: string;
+  status: 'ONLINE' | 'AWAY' | 'OFFLINE';
+  topMsg: string;
+  selected: boolean;
+}
+
+function Chat(props: ChatProps) {
+  const { profileImg, username, status, topMsg, selected } = props;
+  const [isOnHover, setIsOnHover] = useState(false);
+  const [bgColor, setBgColor] = useState('white');
+
+  useEffect(() => {
+    if (selected) {
+      setBgColor('main');
+    }
+    if (isOnHover) {
+      setBgColor('gray100');
+    }
+  }, [isOnHover, selected]);
+  return (
+    <HStack
+      w="260px"
+      h="64px"
+      bgColor={bgColor}
+      p="12px 20px"
+      flex="row"
+      gap="12px"
+      onMouseEnter={() => setIsOnHover(true)}
+      onMouseLeave={() => setIsOnHover(false)}
```

Comment:
```suggestion

  return (
    <HStack
      aria-selected={selected}
      w="260px"
      h="64px"
      p="12px 20px"
      flex="row"
      gap="12px"
      bg="white"
      _hover={{
        bg: 'gray100',
      }}
      _selected={{
        bg: 'main',
      }}
```

이 부분은 [Tabs](https://chakra-ui.com/docs/components/tabs)를 쓰는게 좋을 거 같긴한데 일단 지금은 넘어가도 될거 같아요!